app.js

```diff
@@ -37,7 +37,11 @@ var auth = require('./app/auth/index'),
 if (httpsEnabled) {
      app = express().https({
         key: fs.readFileSync(settings.https.key),
-        cert: fs.readFileSync(settings.https.cert)
+        cert: fs.readFileSync(settings.https.cert),
+        honorCipherOrder: settings.https.honorCipherOrder || true,
```

Comment:
What will happen if someone sets 'false' for this setting? I'm considering just hard coding this as true. This sets whether the server (true) dictates the order of cipher selection or client (false) - we might as well leave it as true and remove the setting option.